keyboards/crkbd/keymaps/ardakilic/keymap.c

```diff
@@ -0,0 +1,482 @@
+/*
+Copyright 2019 @foostan
+Copyright 2020 Drashna Jaelre <@drashna>
+Copyright 2021 Arda Kilicdagi <GH: @ardakilic, TW: @ardadev>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+// Layers
+enum {
+  _QWERTY = 0,
+  _LOWER = 1,
+  _RAISE = 2,
+  _SPACE = 3,
+  _NUMPAD = 4,
+  _SODA = 5,
+  _ADJUST = 6
```

Comment:
The numbers are not needed.  Enums start at 0, and automatically increment. 
```suggestion
  _QWERTY,
  _LOWER,
  _RAISE2,
  _SPACE,
  _NUMPAD,
  _SODA,
  _ADJUST,
```

Reply:
@drashna any reason you've renamed _RAISE to _RAISE2, or was that just a typo ? 

I had problems while compiling due to this and had to specifically set `0` at least to `_QWERTY `back then (using macOS if that would matter), then I thought I'd just put into them all.